packages/app/src/modules/yearOverYear.js

```diff
@@ -3,35 +3,36 @@ import i18n from '@dhis2/d2-i18n'
 // Fixed years generator
 const getFixedYears = len => {
     let year = new Date().getFullYear()
-    return new Array(len)
-        .fill(null)
-        .map(() => ({ id: String(year), name: String(year--) }))
+    return new Array(len).fill(null).map(() => {
+        const yearString = String(year--)
```

Comment:
why a separate `const` instead of just `return { id: String(year), getName: () => String(year--) }`?

Reply:
or is this similar to that other issue recently where you needed a `const` to prevent all items from getting the same name (year)?

Reply:
yep